cpp/tests/ast/transform_tests.cpp

```diff
@@ -108,14 +108,14 @@ TEST_F(TransformTest, NullLiteral)
   auto expression = cudf::ast::operation(cudf::ast::ast_operator::IDENTITY, literal);
 
   auto result   = cudf::compute_column(table, expression);
-  auto expected = column_wrapper<int32_t>({-123, -123, -123, -123}, {0, 0, 0, 0});
+  auto expected = column_wrapper<int32_t>({-123, -123, -123, -123}, {false, false, false, false});
```

Comment:
Can we disable this rule? Writing null masks with 0/1 is much more compact and easier to read, in my opinion.

There are enough of these changes in this PR that I will wait to re-review until we decide on how to go forward with this.

Reply:
Should we disable it globally, or just in tests?

Reply:
Hmm I thought clang-tidy would combine nested configuration files, but that appears not to be the case so never mind. We either enable or disable this globally.

Reply:
I'm ok with disabling it globally.

Reply:
Since this is an autofix based PR I'm going to just reapply all changes from scratch with the rule changed and then force push.

Reply:
OK cool, as expected that trims down the changeset dramatically.